include/Song.h

```diff
@@ -254,6 +254,10 @@ class EXPORT Song : public TrackContainer
 		return m_timeSigModel;
 	}
 
+	void setRecordDelay( int time );
+	int getRecordDelay();
+	void startRecordCountDown();
```

Comment:
`startRecordCountDown` would be private because the countdown should be integrated with `record`.